test/test_torch.py

```diff
@@ -14131,6 +14131,26 @@ def test_cross_device_binary_ops(self, devices):
                 with self.assertRaisesRegex(RuntimeError, "Expected all tensors.+"):
                     op(cpu_tensor, a)
 
+    # This test ensures that a scalar Tensor can be safely used
+    # in a binary operation in conjuction with a Tensor on all
+    # available CUDA devices
+    @deviceCountAtLeast(1)
```

Comment:
This test is pointless if there's a single device?

Reply:
Oh true haha